### PR TITLE
Minor garden cleanups

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

**Comment by mjcarroll** [comment]
@azeey or @iche033 this one is ready

**Comment by mjcarroll** [comment]
It also covers https://github.com/gazebosim/sdformat/pull/1239 and https://github.com/gazebosim/gz-transport/pull/399

File: example/bazel.repos [hunk 1]
```diff
@@ -22,7 +22,7 @@ repositories:
   msgs:
     type: git
     url: https://github.com/gazebosim/gz-msgs
-    version: gz-msgs9
+    version: mjcarroll/garden_bazel
```
Comment (iche033) [inline example/bazel.repos:25]: create a PR for this branch?

**Review by iche033 — APPROVED** [review]

File: example/bazel.repos (hunk 1 above)
Comment (mjcarroll) [reply example/bazel.repos:25]: https://github.com/gazebosim/gz-msgs/pull/406

**Pull request merged**